2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md

```diff
@@ -0,0 +1,124 @@
+  
+These two activities were supported by a health and hygiene promotion component, and a Case Area Targeted Intervention approach at the end of the epidemic.
+  
+MSF also handled one-third of vaccinations in Lilongwe (the capital city has access to 400,000 of the country's stock of one million doses) to assist the Ministry of Health with its vaccination campaign.
+  
+To date, the epidemic is still ongoing although the number of reported cases has declined significantly across the country. A total of 58,500 cases have been confirmed since the beginning of the epidemic, including 1,750 deaths.
+  
+## Cyclone Freddy causing floods and landslides
+  
+In Malawi, the seasonal occurrence of tropical cyclones in February-March has become increasingly apparent, potentially linked to the presence of climate change. The devastating aftermath of Tropical Storm Ana in 2022 prompted [disaster response mapping](https://reliefweb.int/map/malawi/malawi-phalombe-district-cyclone-ana-impact-map-31-january-2022). This year, Tropical Cyclone Freddy made a second landfall in Malawi and Mozambique in March 2023, triggering widespread flooding and devastating landslides.
+  
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog_20230601_malawi_cholera_response.jpeg">
+<p class="caption">The 2nd landfall of Cyclone Freddy caused extensive floods in Phalombe District. Malawi 2023 © MSF
+</p>
+
+**Mapping support for a needs assessment**
+  
+During the Humanitarian Open Mapping Community Working Group meeting in February 2023, Christine Mhone, the representative of OpenStreetMap (OSM) Malawi, presented their OSM community. Jana Bauerová, the Community Engagement Coordinator at MSF, informed in turn about the remote mapping to support the MSF cholera response in Malawi. Subsequent contact and collaboration led to the launch of a HOT Tasking Manager project in March, focused on the Phalombe area, which was of particular interest following the impact of Tropical Cyclone Freddy on March 13, 2023. The project management responsibility for this initiative was assumed by Map Malawi.
+  
+The data gathered through these mapping efforts was used in conducting a needs assessment in Phalombe/Nkulambe, near the border of Malawi and Mozambique, where huge landslides and flash floods heavily damaged entire villages and critical infrastructure. The digitalisation of buildings allowed [MSF](https://www.facebook.com/msfinternational/?__cft__[0]=AZVbT0u2lgB6jfznJ3n1qH9KEptgu_I8GZrPsfuooKh_mRjH9aHCL_bPSSiBFxn_Ka7s11p3YgwyHhk9e9H7PfSxR-3iFgdQfPX8CwU2VJXN-qTIyXx4j9GlWQc9Q9B1_D2FTIYuas7BozT_9JIAWPyuR7UYqExea5TmNMx0RFuWWHcqKB73s7fFkdNj35uG5qY&__tn__=kK-R) to combine the OpenStreetMap (OSM) data with floods and landslide data in maps estimating the impacted zones/buildings (see map cut-out below).
```

Comment:
the link here is just taking me to the MSF organization page on facebook. is that how it's supposed to be working? if so, i think we can use a shorter link?

Reply:
The link opens MSF Canada's Fb page for me, let's go with that for now - I'll double check with Jana on Monday and make changes if needed.